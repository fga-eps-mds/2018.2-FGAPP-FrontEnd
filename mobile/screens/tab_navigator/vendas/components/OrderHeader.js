import React, { Component } from 'react';
import { Image, TouchableHighlight } from 'react-native';
import { Card, CardItem, Text, Left, Right, View } from 'native-base';
import styles from './styles';

class OrderHeader extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    var message = this.props.message;

    return (
      <View style={{padding:10}}>
        <View style={styles.order_header}>
          <View style={{flexDirection: 'column', width: '100%',}}>
            <Text style={styles.order_header_title}>{message}</Text>
          </View>
          <View style={{height: 10}} />
        </View>
      </View>
    );
  }
}
export default OrderHeader;
