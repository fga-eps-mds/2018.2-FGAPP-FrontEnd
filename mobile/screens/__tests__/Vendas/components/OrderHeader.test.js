import React from 'react';
import OrderHeader from '../../../tab_navigator/vendas/components/OrderHeader';
import Adapter from 'enzyme-adapter-react-16';
import {shallow} from 'enzyme';
import Enzyme from 'enzyme';
import renderer from 'react-test-renderer';

Enzyme.configure({adapter: new Adapter()});

it('renders correctly', () => {
  const tree = renderer.create(<OrderHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
