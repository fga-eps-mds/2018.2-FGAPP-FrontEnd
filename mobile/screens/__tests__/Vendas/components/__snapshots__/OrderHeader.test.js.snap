// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
      Object {
        "padding": 10,
      },
    ]
  }
  virtual={undefined}
>
  <View
    style={
      Array [
        Object {},
        Object {
          "borderRadius": 5,
          "flex": 1,
          "flexDirection": "row",
          "paddingBottom": 20,
          "paddingLeft": 20,
          "paddingRight": 20,
          "paddingTop": 10,
        },
      ]
    }
    virtual={undefined}
  >
    <View
      style={
        Array [
          Object {},
          Object {
            "flexDirection": "column",
            "width": "100%",
          },
        ]
      }
      virtual={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#000",
              "fontFamily": "System",
              "fontSize": 16,
            },
            Object {
              "color": "gray",
              "fontSize": 25,
              "textAlign": "center",
            },
          ]
        }
        uppercase={false}
        virtual={undefined}
      />
    </View>
    <View
      style={
        Array [
          Object {},
          Object {
            "height": 10,
          },
        ]
      }
      virtual={undefined}
    />
  </View>
</View>
`;
